test/e2e/pre_provisioning_test.go

```diff
@@ -247,7 +247,7 @@ var _ = ginkgo.Describe("Pre-Provisioned", func() {
 			// Skip these tests until above is fixed.
 			skipIfUsingInTreeVolumePlugin()
 
-			req := makeCreateVolumeReq("pre-provisioned-retain-reclaimPolicy", defaultDiskSize)
+			req := makeCreateVolumeReq("pre-provisioned-retain-reclaim-policy", defaultDiskSize)
```

Comment:
I believe this change is not needed any more since I have pushed the fix for uppercase issue in the crd creation from the provisioners. You can choose to keep or revert these changes.

Reply:
Thanks for addressing this issue, Akash! I think the naming change is more consistent this way, so I am inclined to leave this change.